modules/libcom/src/iocsh/iocsh.cpp

```diff
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <ctype.h>
 #include <errno.h>
+#include <unistd.h>
```

Comment:
Try osiUnistd.h instead of this, it has replacements for most routines when building on Windows. I think this is why your build failed on Windows.